documentation/modules/deploying/proc-deploying-kafkaconnector.adoc

```diff
@@ -30,16 +35,63 @@ spec:
   class: org.apache.kafka.connect.file.FileStreamSourceConnector <3>
   tasksMax: 2 <4>
   config: <5>
+    tasks.max: 1500
     file: "/opt/kafka/LICENSE"
     topic: my-topic
```

Comment:
This was here before your PR. But I think the field name should be `topics`? Did I remembered it right @tombentley? I think it should be in all connectors as well, so it might be good to have a callout for it as well.

Reply:
For the `FileStreamSourceConnector` I think it is `topic` singular (see https://github.com/apache/kafka/blob/trunk/connect/file/src/main/java/org/apache/kafka/connect/file/FileStreamSourceConnector.java#L38), but for _sink_ topics there is also `topics` which names the topics to consume from (https://kafka.apache.org/documentation.html#connect_configuring). Explaining this has got to be worth a digression.

Reply:
Makes sense, thank you. Just a note that we don't actually provide an example sink connector with Strimzi.

Reply:
I meant to say, an example `KafkaConnector` for a sink connector.

Reply:
I've attempted to document both options: `topic` in `FileStreamSourceConnector` and `topics` in `FileStreamSinkConnector`.